tests/system/test_asyncpg_direct_connection.py

```diff
@@ -0,0 +1,152 @@
+# Copyright 2023 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# flake8: noqa: ANN001
+from datetime import datetime
+import os
+
+# fmt: off
+# [START alloydb_native_asyncpg_connect_iam_authn_direct]
+import asyncpg
+
+# [END alloydb_native_asyncpg_connect_iam_authn_direct]
```

Comment:
Shall we split the async pg into a separate file? That might create some duplication, but it will be easier to maintain.